Bhagyashri/Project-WHO/WHO website/css/first.css

```diff
@@ -0,0 +1,92 @@
+body
+{
+	margin:0;
+	padding:0;
+}
+.sec-first{
```

Comment:
Suggestion: try to add proper names for the css classes. sec-first rather user section-first. 
Use of quilified name makes css properties more readable.